TESTING.md

```diff
@@ -16,7 +16,6 @@ be moved from one repository to the next.
 Things to look at:
  - [gtk3-widget-factory](#gtk)
  - [Nautilus](#nautilus)
- - [Gnome shell](#shell)
```

Comment:
Why did you remove this, this links to certain things that should still be checked:
```
    orange underline/straightlines everywhere
    calendar - selected day is orange - days with events are blue
    volume slider should be blue
    active popup menu - orange
    wifi selection is blue
```

Reply:
Because Gnome Shell Theming is handled in a separate repository now.